src/types/app_config.go

```diff
@@ -114,3 +114,18 @@ func (a AppConfig) forEachService(fn func(string, string, ServiceData)) {
 		fn("public", serviceName, data)
 	}
 }
+
+// ValidateProductionFields validates that service.yml contiains a production field
+// and the required fields
+func (a AppConfig) ValidateProductionFields() error {
+	requiredFields := []string{"url", "region", "account-id", "ssl-certificate-arn"}
+	if a.Production == nil {
+		return errors.New("application.yml missing required field 'production'")
```

Comment:
Thoughts on changing "required field" -> "required section" or "required block"?

Reply:
Same for the service config